### ISSUE TITLE
Why use training, validation and test set to build vocabulary in build_data.py ? Isn't it information leak ?

### ISSUE THREAD

**Issue by zhihuacc** [guillaumegenthial/sequence_tagging#94]
I'm not sure but I think including test set to build vocabulary is spilling information from test set to training set. If using test set was OK, then why using only training set to build character set in build_data.py ?
